### ISSUE TITLE
webpack-cli is out-of-date

### ISSUE THREAD

**Issue by wulucxy** [survivejs-demos/webpack-demo#7]
if you run `npm run build`, `webpack-cli` will throw an error:

```javascript
/work/FE/webpack-demo/node_modules/webpack-cli/bin/config-yargs.js:89
				describe: optionsSchema.definitions.output.properties.path.description,
				                                           ^

TypeError: Cannot read property 'properties' of undefined
    at module.exports (/work/FE/webpack-demo/node_modules/webpack-cli/bin/config-yargs.js:89:48)
    at /work/FE/webpack-demo/node_modules/webpack-cli/bin/webpack.js:60:27
    at Object.<anonymous> (/work/FE/webpack-demo/node_modules/webpack-cli/bin/webpack.js:515:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
```

you need to update the newest version。

**Comment by bebraw** [comment]
Which version of npm are you running? I have locked it to known working versions in this project.

I ran `npm upgrade webpack-cli webpack --latest` to try against the latest versions of webpack and webpack-cli and it crashes exactly like that there so something must have broken in webpack itself.

I think I'll write **webpack-cli** out of the demonstration to solve this but some other things have to be done first. Thanks for bringing this up.
